src/main/java/com/uspray/uspray/controller/ClubPrayController.java

```diff
@@ -0,0 +1,74 @@
+package com.uspray.uspray.controller;
+
+import com.uspray.uspray.DTO.ApiResponseDto;
+import com.uspray.uspray.DTO.clubpray.ClubPrayRequestDto;
+import com.uspray.uspray.DTO.clubpray.ClubPrayResponseDto;
+import com.uspray.uspray.DTO.clubpray.ClubPrayUpdateDto;
+import com.uspray.uspray.exception.SuccessStatus;
+import com.uspray.uspray.service.ClubPrayService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/clubpray")
+@Tag(name = "ClubPray", description = "모임 기도제목 API")
+@RequiredArgsConstructor
+@SecurityRequirement(name = "JWT Auth")
+public class ClubPrayController {
+
+    private final ClubPrayService clubPrayService;
+
+    @Operation(summary = "모임 기도제목 생성")
+    @PostMapping
+    public ApiResponseDto<?> createClubPray(@RequestBody ClubPrayRequestDto clubPrayRequestDto,
+        @Parameter(hidden = true) @AuthenticationPrincipal User user) {
+        clubPrayService.createClubPray(clubPrayRequestDto, user.getUsername());
+        return ApiResponseDto.success(SuccessStatus.CREATE_CLUB_PRAY_SUCCESS,
+            SuccessStatus.CREATE_CLUB_PRAY_SUCCESS.getMessage());
+    }
+
+    @Operation(summary = "모임 기도제목 수정")
+    @PostMapping("/update")
```

Comment:
```suggestion
    @PutMapping
```
수정을 ['POST'], "/update" 로 하신 이유가 궁금합니다.
API의 행위는 method로 나타내라 배웠는데
['PUT']을 사용하고 url을 post와 통일하는 게 어떨까요?